packages/block-library/src/social-links/edit.js

```diff
@@ -58,23 +59,33 @@ export function SocialLinksEdit( props ) {
 
 	const {
 		iconBackgroundColorValue,
+		customIconBackgroundColor,
 		iconColorValue,
 		openInNewTab,
 		size,
 		layout,
 	} = attributes;
+
 	const usedLayout = layout || getDefaultBlockLayout( name );
 
 	// Remove icon background color if logos only style selected.
```

Comment:
```suggestion
	// Remove icon background color when logos only style is selected or
	// restore it when any other style is selected.
```
Just to mention the added behavior in the comment. I also think this comment would go better on the lines just before the the effect hook but I suppose it's okay where we found it.